hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionServer.java

```diff
@@ -1611,9 +1611,13 @@ protected void initializeMemStoreChunkCreator() {
       float initialCountPercentage = conf.getFloat(MemStoreLAB.CHUNK_POOL_INITIALSIZE_KEY,
           MemStoreLAB.POOL_INITIAL_SIZE_DEFAULT);
       int chunkSize = conf.getInt(MemStoreLAB.CHUNK_SIZE_KEY, MemStoreLAB.CHUNK_SIZE_DEFAULT);
+      float indexChunkPercentage = conf.getFloat(MemStoreLAB.INDEX_CHUNK_PERCENTAGE_KEY,
+          MemStoreLAB.INDEX_CHUNK_PERCENTAGE_DEFAULT);
```

Comment:
So now if ARRAY_MAP is used, the user has to make the INDEX_CHUNK+PERCENTAGE as 0 instead of using the defualt 0.1? May be we should document this then.

Reply:
OK, has document this on MemStoreLAB#INDEX_CHUNK_PERCENTAGE_KEY